examples/tutorial/basic-5/programs/basic-5/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "basic-5"
+version = "0.1.0"
+description = "Created with Anchor"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "basic_5"
+
+[features]
+no-entrypoint = []
+no-idl = []
+no-log-ix-name = []
+cpi = ["no-entrypoint"]
+default = []
+
+[dependencies]
+anchor-lang = "0.26.0"
```

Comment:
Use dependency to the repository version instead of a released and outdated version.

Reply:
Done.